src/main/java/edu/harvard/iq/dataverse/engine/command/impl/FinalizeDatasetPublicationCommand.java

```diff
@@ -224,6 +227,20 @@ public boolean onSuccess(CommandContext ctxt, Object r) {
             LoggingUtil.writeOnSuccessFailureLog(this, failureLogText,  dataset);
             retVal = false;
         }
+        
+        //re-indexing dataverses that have additional subjects
+        if (!dataversesToIndex.isEmpty()){
+            for (Dataverse dv : dataversesToIndex) {
+                try {
+                    Future<String> indexString = ctxt.index().indexDataverse(dv);
+                } catch (IOException | SolrServerException e) {
```

Comment:
Maybe a stupid question - but is it wrong to just "catch (Exception e)" in a situation like this - where we don't care why it failed/what kind of an exception it was? Does catching any exception make it more reliable potentially... or are there reasons why it should be avoided?

Reply:
In general, but maybe not in this case, if we get an exception that is unexpected we want it to fail loudly so that we can address it in another way if necessary.

Reply:
OK, makes sense.